k8s/README.md

```diff
@@ -0,0 +1,13 @@
+## Kubernetes Files for Versions 2.1.1 and Up
+
+Detailed instructions for installing and upgrading the Scalyr agent in kubernetes can be found [here](https://app.scalyr.com/help/install-agent-kubernetes).
```

Comment:
I try to be consistent with capitalizing both the S and A in _Scalyr Agent_.  Please take a pass at doing that.  Also, the K in Kubernetes should be capitalized.  Please do that here and in the other README.md file you created.